app/views/battles/show.html.haml

```diff
@@ -13,7 +13,7 @@
       - if @battle.won?
         .columns.small-2
           %strong WINNER:
-          = image_tag @battle.winning_comment.image_url
+          = text_with_images @battle.winning_comment.image, true
```

Comment:
Do we need to pass ```true``` as a second argument here, given that it is the default value for the ```flex``` parameter in ```text_with_images```